actions/hybrid_branch_deploy/action.yml

```diff
@@ -66,6 +69,8 @@ runs:
           branch=${{ github.head_ref }}
         cache-from: type=gha
         cache-to: type=gha,mode=max
+      env:
+        METRONOME_PAT: ${{ inputs.metronome_pat }}
```

Comment:
Looks good! Just wanted to add a thought on the naming convention here.... How about we rename this like metronome_access_token or metronome_secret_key etc. or even something else ? Something that could be intuitive.

Reply:
sure I can do that